### PR TITLE
bump docker to v1.10.0

### PR DESCRIPTION
**Pull request by cmosetick**

### CODE REVIEW

**Comment by bigkraig** [comment]
+1

File: Dockerfile [hunk 1]
```diff
@@ -23,7 +23,7 @@ ENV BUILDROOT_VERSION 2015.11.1
 RUN wget -qO- http://buildroot.uclibc.org/downloads/buildroot-$BUILDROOT_VERSION.tar.bz2 | tar xj && \
     mv buildroot-$BUILDROOT_VERSION /tmp/buildroot
 
-ENV DOCKER_VERSION 1.9.1
+ENV DOCKER_VERSION 1.10
```
Comment (nlf) [inline Dockerfile]: this needs to be 1.10.0, with it like this the download is a 404

**Comment by bigkraig** [comment]
@cmosetick fixit! =)

**Comment by cmosetick** [comment]
@bigkraig I've added the missing 0!!
@nlf Can you merge?

**Pull request merged**